domains/transfer/Transfer.API/API.IntegrationTests/SwaggerTests.GetSwaggerDocs_v1.0.verified.txt

```diff
@@ -0,0 +1,265 @@
+﻿{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Transfer API 1.0",
+    "description": "",
+    "version": "1.0"
+  },
+  "paths": {
+    "/api/transfer-agreement-proposals": {
+      "post": {
+        "tags": [
+          "TransferAgreementProposal"
+        ],
+        "summary": "Create TransferAgreementProposal",
+        "parameters": [
+          {
+            "name": "EO_API_VERSION",
+            "in": "header",
+            "description": "The requested API version",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "default": "1.0"
```

Comment:
default version should be 20230101

Reply:
in v1.0? How do I set it to default 20230101?

Reply:
deleted the verified as it was not needed